src/components/common/ListItem.js

```diff
@@ -0,0 +1,119 @@
+// @flow
+
+import React from 'react';
+import { View, Text, TouchableOpacity, Image } from 'react-native';
+import { MediaQueryStyleSheet } from 'react-native-responsive';
+
+import GlobalStyles from '../../global/Styles';
+import AssetsImages from '../../global/AssetsImages';
+
+type Props = {
+  /**
+   * @desc Text to display on item
+   * @type string
+   */
+  text?: string,
+  /**
+   * @desc Style object for basic text style
+   * @type object
+   */
+  textStyle?: any,
+  /**
+   * @desc Text to show under main text.
+   */
+  subtitle?: string | null,
+  /**
+   * @desc Additional view to render on the right of the list item.
+   * @type string
+   */
+  AdditionalView: any,
+  /**
+   * @desc Flag to show/hide disclosure indicator.
```

Comment:
same goes for `textStyle` and `AdditionalView`

Reply:
Fixed.